gapis/replay/builder/builder.go

```diff
@@ -49,7 +50,12 @@ type marker struct {
 // ResponseDecoder decodes all postback responses from the replay virtual machine.
 // If err is nil, then r is the Reader to the sequential postback data. If r is nil,
 // then the postback data was absent or corrupted and err holds the error.
-type ResponseDecoder func(r io.Reader, err error)
+type ResponseDecoder func(p *gapir.PostData)
+
+type postBackDecoder struct {
+	expectedSize int
+	decode       Postback
+}
 
 // Postback decodes a single commands's postback, returning and carrying over errors.
```

Comment:
I think we should remove the `error` return value from this function signature as it is completely unused, and will simplify the implementations.
Please also update docs.

Reply:
Done, also changed a lot of Post() caller sites in gapis/api/gles/find_issues.go.